app/views/candidate_interface/application_form/review_previous_application.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for :title, t('page_titles.submitted_application') %>
+
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path) %>
+
+<h1 class="govuk-heading-xl">
+  <%= t('page_titles.unsubmitted_previous_application') %>
```

Comment:
Can we make sure this reflects whether the application is submitted or not. Suggest wrapping this in an `if submitted_at_date.present?`, and choosing between the submitted and unsubmitted titles in there. Similarly for the `content_for :title` above

Reply:
Sure thing, ill put it in now